Problem300/Python/solution_1.py

```diff
@@ -0,0 +1,3 @@
+
+if __name__ == "__main__":
```

Comment:
and this? only testing?

Reply:
Yes I commented problem 501 and added 300 / 301 for testing

Reply:
ok